src/global/reco/ParticleFlow_factory.h

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) Derek Anderson
+
+#pragma once
+
+#include <map>
+#include <vector>
+#include <utility>
+#include <spdlog/logger.h>
+#include <extensions/spdlog/SpdlogMixin.h>
+#include <extensions/jana/JChainMultifactoryT.h>
+// event data model definitions
+#include <edm4eic/ReconstructedParticleCollection.h>
+// for geometry services
+#include <services/geometry/dd4hep/DD4hep_service.h>
+// necessary algorithms and configurations
+#include "algorithms/reco/ParticleFlow.h"
+#include "algorithms/reco/ParticleFlowConfig.h"
+
+namespace eicrecon {
+
+  class ParticleFlow_factory :
+          public JChainMultifactoryT<ParticleFlowConfig>,
```

Comment:
Please use JOmniFactory for any new algorithms. Good example is src/factories/calorimetry/CalorimeterClusterRecoCoG.